lib/utils/requireModule.js

```diff
@@ -1,12 +1,15 @@
 const path = require('path');
 
-const requireModule = module.exports = function(fullpath) {
+const requireModule = module.exports = function(fullpath, error={}) {
   let exported;
   try {
+    if (Object.keys(error).length > 0) {
+      throw new Error(`${error.code}: ${error.message}`)
+    }
     exported = require(fullpath);
   } catch (err) {
     if (err.code === 'MODULE_NOT_FOUND') {
-      return requireModule(path.resolve(fullpath));
+      return requireModule(path.resolve(fullpath), {code: err.code, message: err.message});
```

Comment:
seems like you could just pass the `err` object here.

Reply:
fixed in [`4d6b137`](https://github.com/nightwatchjs/nightwatch/pull/3123/commits/4d6b1370eb894654388a2218c34a91b2c5d7b526)